apps/condo/domains/meter/schema/Meter.js

```diff
@@ -97,6 +97,38 @@ const Meter = new GQLListSchema('Meter', {
             schemaDoc: 'Client\'s billing account',
             type: Text,
             isRequired: true,
+            hooks: {
+                validateInput: async ({ context, operation, existingItem, resolvedData, fieldPath, addFieldValidationError }) => {
+                    const value = resolvedData[fieldPath]
+                    let metersWithSameAccountNumberInOrganization
+                    if (operation === 'create') {
+                        metersWithSameAccountNumberInOrganization = await MeterApi.getAll(context, {
+                            accountNumber: value,
+                            organization: { id: resolvedData.organization },
+                            deletedAt: null,
+                            OR: [
+                                { unitName_not: value },
+                                { property: { id_not: resolvedData.property } },
```

Comment:
is the resolvedData.property exists if we don't update the property field?

Reply:
it's create operation, property always exist